src/server/config/environment/development.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
   apiUrl: 'http://localhost:3001',
   apiInternalUrl: 'http://api:3001',
   landingUrl: 'http://localhost:3000',
+  websocketsUrl: 'http://localhost:8082',
```

Comment:
I think it should be `webSocketUrl` (camelCase and in the singular)

Reply:
Ok